test/check-subscriptions

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3 -cimport os, sys; os.execv(os.path.dirname(sys.argv[1]) + "/../test/common/pywrap", sys.argv)
+#!/usr/bin/python3 -cimport os, sys; os.execv(os.path.dirname(sys.argv[1]) + "/common/pywrap", sys.argv)
```

Comment:
Are there tests run in subscription-manager itself? Because that might be the reason its a relative path?

Reply:
hmm I think I dont understand the difference between `../test/common/` or just directly doing `/common/` in this case